assembly.html.markdown

```diff
@@ -0,0 +1,218 @@
+---
+language: assembly
+filename: learn.asm
+contributors:
+    - ["", ""]
+---
+
+Assembly language is a low-level programming language for a computer
+
+```asm
+; Comments
+
+section .data
+; it contains data, constants
```

Comment:
`s/it/It/`. Also could you give a few examples of C code that would generate this section? (String literals, etc.)